core/domain/interaction_jobs_one_off_test.py

```diff
@@ -755,6 +763,205 @@ def test_no_action_is_performed_for_deleted_exploration(self):
+                'html': '<p>This is value1 for MultipleChoiceInput</p>',
+                'content_id': 'ca_choices_0'
+            }, {
+                'html': '<p>This is value2 for MultipleChoiceInput</p>',
+                'content_id': 'ca_choices_1'
+            }, {
+                'html': '<p>This is value3 for MultipleChoiceInput</p>',
+                'content_id': 'ca_choices_2'
+            }, {
+                'html': '<p>This is value4 for MultipleChoiceInput</p>',
+                'content_id': 'ca_choices_3'
+            }]},
+            'showChoicesInShuffledOrder': {'value': True}
+        }
+
+        state2.update_interaction_id('MultipleChoiceInput')
+        state2.update_interaction_customization_args(customization_args_dict2)
+        state2.update_next_content_id_index(4)
+
+        exp_services.save_new_exploration(self.albert_id, exploration)
+
+        # Start MultipleChoiceInteractionLimitOneOffJob job
+        # on sample exploration whose interaction is
+        # Multiple Choice Input Interaction.
+        job_id = start_multiple_item_interaction_lt_one_off_job(
+            self, interaction_jobs_one_off.MultipleItemInteractionLtOneOffJob)
+
+        actual_output = (
+            interaction_jobs_one_off
+            .MultipleItemInteractionLtOneOffJob.get_output(job_id))
+        expected_output = ('LONGER THAN 30', 38)
+
+        self.assertEqual(actual_output, expected_output)
+
+        state3.update_interaction_id('ItemSelectionInput')
+        state3.update_interaction_customization_args(customization_args_dict1)
+        state3.update_next_content_id_index(6)
+
+        exp_services.save_new_exploration(self.albert_id, exploration)
+
+        # Start MultipleItemInputInteractionLtOneOffJob job
+        # on sample exploration whose interaction is
+        # Item Selection Input.
+        job_id = start_multiple_item_interaction_lt_one_off_job(
+            self, interaction_jobs_one_off.MultipleItemInteractionLtOneOffJob)
+
+        actual_output = (
+            interaction_jobs_one_off
+            .MultipleInputInteractionLtOneOffJob.get_output(job_id))
+        expected_output = ('SUCCESS', 1)
+        self.assertEqual(actual_output, expected_output)
+
+        state4.update_interaction_id('ItemSelectionInput')
+        state4.update_interaction_customization_args(customization_args_dict2)
+        state4.update_next_content_id_index(8)
+
+        exp_services.save_new_exploration(self.albert_id, exploration)
+
+        # Start MultipleItemInputInteractionLtOneOffJob job
+        # on sample exploration whose interaction is
+        # Item Selection Input.
+        job_id = start_multiple_item_interaction_lt_one_off_job(
+            self, interaction_jobs_one_off.MultipleItemInteractionLtOneOffJob)
+
+        actual_output = (
+            interaction_jobs_one_off
+            .MultipleInputInteractionLtOneOffJob.get_output(job_id))
+        expected_output = ('LONGER THAN 30', 38)
```

Comment:
@vojtechjelinek Can you please help me how to write the expected output in unicode because I am a little weak in that.
('LONGER THAN 30', (item.id, choice_length)) .I think this is the code which would come to the expected output.
Thanks!

Reply:
I think if you rework the run method according to https://github.com/oppia/oppia/blob/develop/core/domain/activity_jobs_one_off_test.py#L76 you should be able to compare the exact output.